src/metalperformanceshaders.cs

```diff
@@ -695,15 +895,58 @@ interface MPSCnnConvolutionDescriptor : NSCopying, NSSecureCoding {
 		[Static]
 		[Export ("cnnConvolutionDescriptorWithKernelWidth:kernelHeight:inputFeatureChannels:outputFeatureChannels:neuronFilter:")]
 		MPSCnnConvolutionDescriptor GetConvolutionDescriptor (nuint kernelWidth, nuint kernelHeight, nuint inputFeatureChannels, nuint outputFeatureChannels, [NullAllowed] MPSCnnNeuron neuronFilter);
+
+		[TV (11, 0), iOS (11, 0)]
+		[Static]
+		[Export ("supportsSecureCoding")]
+		bool SupportsSecureCoding { get; }
+
+		[TV (11, 0), iOS (11, 0)]
+		[Static]
+		[Export ("cnnConvolutionDescriptorWithKernelWidth:kernelHeight:inputFeatureChannels:outputFeatureChannels:")]
+		MPSCnnConvolutionDescriptor CreateCnnConvolutionDescriptor (nuint kernelWidth, nuint kernelHeight, nuint inputFeatureChannels, nuint outputFeatureChannels);
+
+		[TV (11, 0), iOS (11, 0)]
+		[Export ("setBatchNormalizationParametersForInferenceWithMean:variance:gamma:beta:epsilon:")]
+		void SetBatchNormalizationParameters (IntPtr /* float* */ mean, IntPtr /* float* */ variance, [NullAllowed] IntPtr /* float* */ gamma, [NullAllowed] IntPtr /* float* */ beta, float epsilon);
```

Comment:
`[Internal]` maybe? You have a code-behind implementation for this, which should be sufficient I think.